core/templates/dev/head/domain/collection/SearchExplorationsBackendApiServiceSpec.js

```diff
@@ -0,0 +1,57 @@
+// Copyright 2015 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests for SearchExplorationsBackendApiService.
+ */
+
+describe('Exploration search backend API service', function() {
```

Comment:
Good start on tests, but to ensure sufficient coverage, we would want tests to cover the case were it does return items and one where the server returns a 500 error.